test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -251,13 +247,11 @@ describe('HTTP: Measure reception ', function() {
         beforeEach(function(done) {
             nock.cleanAll();
 
-            // Note /v1/updateContext response is not processed by IOTA so its content is irrelevant,
-            // as far as it is a 200 OK
             contextBrokerMock = nock('http://192.168.1.1:1026')
                 .matchHeader('fiware-service', 'smartGondor')
                 .matchHeader('fiware-servicepath', '/gardens')
-                .post('/v1/updateContext')
-                .reply(200, '{}');
+                .post('/v2/entities?options=upsert')
```

Comment:
Similar to what we have discussed in https://github.com/telefonicaid/iotagent-node-lib/pull/615#discussion_r198862320 maybe the usages of post in mock without payload in this PR should include a comment explaining the situation.

Reply:
Fixed with 787d2ae2e19ab95c27e5872db5330d4b6c22d573